rules-topics/15_smithing.md

```diff
@@ -0,0 +1,99 @@
+#### Pick Pocket Tools
+   >These tools require the Cutpurse skill to use. These tools allow the would-be thief to slip a hand into another character pouch or pocket and randomly take a single game item. This item is represented by a clip that has an ID that corresponds to the character. This ID is issued by Logistics, though a player can request a new ID at any time. The player slips the clip onto the pouch, or pocket, and goes to notify a Game Marshal. The Game Marshal will approach the picked character and retrieve one random game item from the picked character. If there is nothing in the picked pouch, then the larceny has resulted in no reward. The transfer of goods from Game Marshal to thief takes place out of sight from other players.  Tools have 10 charges.
+#### Ranged Weapon
+   >This creates a finely made Ranged Weapon that has a base damage of 1, with 5 Durability. 
+   
+### Tier VII
+
+#### Resistant Plating 
+   >This armor plating is specially designed to help absorb an otherwise direct blow, shrugging off the effects at the cost of the brittle material. This plating can be applied to a suit of armor with 60 seconds of roleplay. Depending on how the user chooses to configure it upon application, it can grant them a single use of Resist Physical or Spell. A suit of armor can only have one Resistant Plating applied at a time. This takes a defense slot. This item is consumed on use.
+#### Reinforcement Kit
+>This kit helps prepare the user for imminent destruction by providing them with several small tools that when quickly applied can help prevent an item from being destroyed. This can be prepared with 1 Minute of roleplay. It grants the user a single use of Resist Destruction, that lasts until Rest. An individual may only have one Reinforcement Kit active on it at time. This item is consumed on use. This does not take a defense slot.
+#### Weighted Ammunition
+   >This specially designed item is made from very dense materials, and is made to blow past enemy defenses to really hit them where it hurts. This ammunition can either be fired from a Ranged Weapon or used as a Thrown Weapon. While it carries a heavy impact, the design means that its unwieldy nature makes it difficult to use with precision. Using this ammunition will deal 5 Normal Vital damage, this damage call cannot be modified any means. This item is consumed on use.
+
+### Tier VIII
+
+#### Expandable Weapon
+> A marvel of intricate crafting, this weapon modification is designed to allow the wielder the ultimate flexibility in controlling their reach. With 60 seconds of roleplay, the wielder may change the length of this weapon, from up to a Two-Handed weapon down to a Short length one-handed weapon. This allows the wielder to use this item with a Short, Long, or Two-Handed weapon rep interchangably.
```

Comment:
`60 seconds` to `1 minute` here, please.

Clarification request on this one before it happens:  Use with unarmed/claws?